client/src/backend/remote_node.rs

```diff
@@ -240,9 +240,11 @@ impl backend::Backend for RemoteNode {
 }
 
 async fn check_runtime_version(rpc: &Rpc) -> Result<(), Error> {
-    match runtime_version(rpc, None).await?.spec_version {
-        9 | 10 | 11 | 12 | 13 | 14 => Ok(()),
-        other => Err(Error::IncompatibleRuntimeVersion(other)),
+    let spec_version = runtime_version(rpc, None).await?.spec_version;
+    if spec_version == VERSION.spec_version {
+        Ok(())
+    } else {
+        Err(Error::IncompatibleRuntimeVersion(spec_version))
```

Comment:
I think that keeping the `match` is more welcome for future us trying to extend the list of supported versions:
```suggestion
    const CURRENT_SPEC_VERSION: u32 = VERSION.spec_version;
    match runtime_version(rpc, None).await?.spec_version {
        CURRENT_SPEC_VERSION => Ok(()),
        other => Err(Error::IncompatibleRuntimeVersion(other)),
```
Of course if we want to drop the explicit list of versions. I'm concerned that now it will be dangerously easy to accidentally remove perfectly fine support for the previous version if we introduce a new one.

Reply:
This is great actually. I wasn’t aware that `VERSION` was constant and that you could easily get the field.